_includes/manuals/1.15/4.1.2_roles_and_permissions.md

```diff
@@ -12,7 +12,9 @@ These may be created, deleted and edited on the **Roles** page. Each role will c
 
 There is one built-in system role, 'Default role'. This is a set of permissions that every user will be granted, in addition to any other roles that they have. Foreman provides you with a set of seeded roles. These roles can be assigned to users but cannot be modified in any way. They serve as a sane set of defaults and a quick starting point. If you wish to base your custom role on one of these, you can clone it and modify the clone.
 
-Roles can be also associated to Locations or Organizations if these are allowed. Unlike other objects this does not mean that Roles would be only available in a particular scope. Roles are always global for the whole Foreman. The association means that filters of such role are scoped to a particular Organization or Location. Imagine you want to create a role representing Manager of Organization A. You can clone an existing Manager role and associate it with Organization A. If you later assign this role to some users, they will be granted manager permissions but only on resources of Organization A. Note that some resource are not scopeable by Organization and Locations. Filters for such resources grant permissions globally.
+Roles can be also associated to Locations or Organizations if these are allowed. Unlike other objects this does not mean that Roles would be only available in a particular scope. Roles are always global for the entirety of Foreman. The association means that filters of such role are scoped to a particular Organization or Location. Imagine you want to create a role representing Administrator of Organization A. You can clone an existing Organization admin role and associate it with Organization A. If you later assign this role to some users, they will be granted all admin permissions but only on resources of Organization A. Note that some resource are not scopeable by Organization and Locations. Filters for such resources grant permissions globally.
```

Comment:
Nitpick - Hard to comment on very long lines (consider re-wrapping at 80 chars :P) but you've got an instance of "resource" instead of "resources" on the second-to-last line "Note that some resource ..."